crates/package_json/src/lib.rs

```diff
@@ -16,6 +17,22 @@ pub struct PackageJson {
     value: Value,
 }
 
+pub enum DependencyType {
```

Comment:
Should this be called DependencyGroup instead? Should I add peer dependencies, bundled dependencies in this PR?

Reply:
I like dependencygroup better, but I'm ok with dependencytype as well. I think we can split the PRs. That's ok